packages/application-components/src/components/dialogs/info-dialog/info-dialog.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CloseIcon,
+  Constraints,
+  SecondaryIconButton,
+  Spacings,
+  Text,
+} from '@commercetools-frontend/ui-kit';
+import ModalContainer from '../modal-container';
+import styles from './info-dialog.mod.css';
+
+// Like "Spacings.Inline", but with the `justify-content: space-between` style
+const Stretched = props => (
+  <div className={styles.stretched}>{props.children}</div>
+);
+Stretched.displayName = 'Stretched';
+Stretched.propTypes = { children: PropTypes.node.isRequired };
+
+const InfoDialog = props => (
+  <ModalContainer
+    isOpen={props.isOpen}
+    closeTimeoutMS={props.closeTimeoutMS}
+    onRequestClose={props.onClose}
+    overlayClassName={styles['modal-overlay']}
+    className={styles['modal-content']}
+    contentLabel="info-dialog"
+  >
+    <Constraints.Horizontal constraint={props.horizontalConstraint}>
+      <Card>
+        <Spacings.Stack scale="m">
+          <Stretched>
+            <Text.Subheadline
+              elementType="h4"
+              truncate={true}
+              title={typeof props.title === 'string' ? props.title : undefined}
```

Comment:
Why do we need this check?

Reply:
The `props.title` can be any react node, whereas the `Text.title` needs to be a string (it's the HTML `title` attribute)

Reply:
I think it would be cleaner to just force people to pass in strings.